File: PRAK404-2310817220016-NailaHanifah.py
while True:
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")

    try:
        pilihan = int(input("Masukkan Pilihan: "))
    except ValueError:
        print("Input harus berupa angka. Silakan coba lagi.")
        continue

    if 1 <= pilihan <= 4:
        try:
            nilai_pertama = float(input("Masukkan nilai pertama: "))
            nilai_kedua = float(input("Masukkan nilai kedua: "))
        except ValueError:
            print("Input harus berupa angka. Silakan coba lagi.")
            continue

        if pilihan == 1:
            hasil = nilai_pertama + nilai_kedua
        elif pilihan == 2:
            hasil = nilai_pertama - nilai_kedua
        elif pilihan == 3:
            hasil = nilai_pertama * nilai_kedua
        elif pilihan == 4:
            if nilai_kedua != 0:
                hasil = nilai_pertama / nilai_kedua
            else:
                print("Pembagian oleh nol tidak valid.")
                continue

        print(f"Hasil {pilihan} antara {nilai_pertama:.2f} dan {nilai_kedua:.2f} adalah {hasil:.2f}")
    elif pilihan == 5:
        print("Terimakasih, telah menggunakan kalkulator Naila Hanifah")
        break
    else:
        print("Input anda salah, silakan coba lagi.")

File: PRAK404-2310817220016-NailaHanifah.c
#include <stdio.h>

int main() {
    int pilihan;
    float nilaiPertama, nilaiKedua;

    do {
        printf("Pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan: ");
        scanf("%d", &pilihan);

        if (pilihan >= 1 && pilihan <= 4) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &nilaiPertama);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &nilaiKedua);

            switch (pilihan) {
                case 1:
                    printf("Hasil penjumlahan antara %.2f dan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, nilaiPertama + nilaiKedua);
                    break;
                case 2:
                    printf("Hasil pengurangan antara %.2f dan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, nilaiPertama - nilaiKedua);
                    break;
                case 3:
                    printf("Hasil perkalian antara %.2f dan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, nilaiPertama * nilaiKedua);
                    break;
                case 4:
                    if (nilaiKedua != 0) {
                        printf("Hasil pembagian antara %.2f dan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, nilaiPertama / nilaiKedua);
                    } else {
                        printf("Pembagian oleh nol tidak valid.\n");
                    }
                    break;
            }
        } else if (pilihan == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Naila Hanifah\n");
        } else {
            printf("Input anda salah, silahkan coba lagi\n");
        }
    } while (pilihan != 5);

    return 0;
}